Assets/Scripts/Cards/ChaosCard.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Chaos Power Card", menuName = "Chaos Power Card")]
+public class ChaosCard:Card {
+
+	private override bool targetable = true;
+	private override bool canTarget = false; // Can't target, can only defend
```

Comment:
This variable name is good but I'm wondering what the `override` keyword does here? Did the compiler tell you to put this in?

Reply:
Woops yeah that shouldn't be there at all (was a total guess on my part, forgot to test it in the editor)